### DISCUSSION TITLE
[Scam Alert]  BEMCHAIN بِم چین

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#37] [Announcements]
<div dir="rtl">

**Project Name:** 
BEMCHAIN

**نام پروژه:** 
بِم چین

**نماد پروژه:**
bemt, bcn

**اتفاقی که افتاده است:**

پروژه BEMT با ادرس قرارداد https://tronscan.io/#/token20/TBp6ZMzkxci5o4sJjFa6Fo9Wy36gcubQLW

نزدیک به 70درصد حجم توکن ها در اختیار یه تعداد به خصوصی(50 نفر) قرار دارد که مطمینا از تعداد موجود در کیف پول میشه متوجه شد که متعلق به خود شرکته. اکثر هولدرا فیک هستند و به صورت یکجا توی یک بازه زمانی مشخص انتقال انجام شده به حسابشون

![photo_2021-09-15_15-44-31](https://user-images.githubusercontent.com/87842536/133969896-d9a9190e-e75b-456f-bf07-b446eb5005cf.jpg)
![photo_2021-09-15_15-44-31 (2)](https://user-images.githubusercontent.com/87842536/133969955-c1471059-4ba5-4bf6-b26a-0a83e076a48a.jpg)

کل مبلغ دلاری موجود در لیکوییدی این شرکت تنها 15هزار دلاره . با یک ضرب و تقسیم معمولی از توکن های موجود داخل لیکوییدیتی میشه متوجه شد

![photo_2021-09-20_12-13-27](https://user-images.githubusercontent.com/87842536/133970263-a61def52-9f1a-4fb7-94a5-bcd5a2ce5be8.jpg)

نحوه خرید توکن و زیر مجموعه گیری

![photo_2021-09-20_12-13-39](https://user-images.githubusercontent.com/87842536/133970309-d698cdf9-a210-45d0-b95e-e3433ac9a354.jpg)

پلن ها 🤩 در شبکه گیری عالین حتما استفاده کنید

![photo_2021-09-20_12-15-50](https://user-images.githubusercontent.com/87842536/133970500-e47c4626-7f96-467a-bcac-a6384d07b467.jpg)
گیت هاب هم جدیدا certıfıcate میده🤣

![photo_2021-09-20_12-16-02](https://user-images.githubusercontent.com/87842536/133970528-3e121b02-affd-4c86-afbc-3748d2b1155e.jpg)
اونم به شرکتی که کلا یک پوش داشته اونم تبلیغات خودش رو گذاشته

![photo_2021-09-20_12-17-11](https://user-images.githubusercontent.com/87842536/133970632-ad1a4f28-8240-424d-a2a9-e108d224b5a6.jpg)

رودمپ اوضاعش بدتره . نه audıt شده نه جایی لیست شده . تازه قرار بوده کازینو بزنن که ظاهرا برنامه داره عوض میشه

![photo_2021-09-20_12-17-22](https://user-images.githubusercontent.com/87842536/133970643-88767da8-46a5-4f40-8c93-92f9e9eb6956.jpg)

قیمت قبل از فروش اصلی که در وایت پیپر گفتن لیست شده و خرید و فروش میشده . پامپ از یک سنت تا 400 دلار تنها با چندصد دلار در کمتر از یک روز .

روش سوددهی هم تاجایی که من فهمیدم هم با ریفرال گیری و هم با هولد و ماین شت کویناشون

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار هرمی

**نوع کلاهبرداری:** 
طرح هرمی، طرح پانزی

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**

**آدرس وب سایت(ها):**
http://bemchain.io
http://beminfo.io 
https://t.me/bemtokeninfo
https://t.me/bemtpersian
https://bemchain.com/

**برآورد مبلغ کلاهبرداری:**

**منابع:**

**جمع بندی:**

**Comment by saeedkousha** [comment]
با سلام 
روش کار به این صورت است که در ابتدا 
خرید بم توکن انجام میشود و پس از ایجاد لینک ریفرال توکن بی سی ان دریافت میگردد 
سپس هر شخص دو زیر مجموعه گرفته و مراحل به همین صورت تکرار میشود.
